### DISCUSSION TITLE
CubeFS Tutorial including Authnode

### DISCUSSION THREAD

**Discussion by Zorlin** [cubefs/cubefs#3090] [Show and tell]
Hey folks!

I made a tutorial! It goes over the complete setup of a CubeFS cluster including setting up the Authnode cluster and setting up full authentication.

A future article will cover the Object node and setting it up, as well as setting up the Blobnode and erasure coding, maybe a series on monitoring... but this is the first complete documentation in English I've seen for setting up CubeFS with authentication :) Check it out and let me know what you think!

Thanks for CubeFS!

**Comment by Zorlin** [comment]
The tutorial is at https://raptorswithhats.com/cubefs/ :) It's a blog with no tracking or ads.

**Reply by xiaochunhe** [reply]
good job，thank you :）
